### ISSUE TITLE
Stylelint Invalid Option value for "ignore" of rule "scss/media-feature-value-dollar-variable"

### ISSUE THREAD

**Issue by rgant** [stylelint-scss/stylelint-scss#1105]
Stylelint (16.13.0) reports an error:

> Invalid Option: Invalid value "no-preference" for option "ignore" of rule "scss/media-feature-value-dollar-variable"

For config:

```json
    "scss/media-feature-value-dollar-variable": [
      "always",
      { "ignore": ["no-preference"] }
    ],
```

Similar for `"ignore": ["dark"]`:

> Invalid Option: Invalid value "dark" for option "ignore" of rule "scss/media-feature-value-dollar-variable"

However it doesn't report an Invalid Option for:

```json
    "scss/media-feature-value-dollar-variable": [
      "always",
      { "ignore": ["keywords"] }
    ],
```

**Comment by kristerkari** [comment]
Thanks! I'll need to have a look at this.

**Comment by Mouvedia** [comment]
It seems like the `ignore` option only accept `"keywords"`.
cf https://github.com/stylelint-scss/stylelint-scss/commit/4421f059d3f524e1e3d212d58e662f811ce4ece7#diff-804619abf0970a3cecd5304600008c762d6da20c82791ad7d4b22e80b008f8c1R90

**Comment by kristerkari** [comment]
> It seems like the `ignore` option only accept `"keywords"`.

Ah right, it is not a list of keywords, but instead the string "keywords". Thanks for pointing it out! 👍
